scripts/test/diff_dashboard_states.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Requirements:
+# - Imagemagick installed with PNG support locally
+
+BRANCH1=$(git rev-parse --abbrev-ref HEAD)
+BRANCH2=$1
+
+shift 1
+
+# verify the branch exists
+if ! git rev-parse --verify $BRANCH2 > /dev/null
+then
+  echo "$BRANCH2 is not a valid branch"
+  exit 1
+fi
+
+if ! type "magick" > /dev/null
+then
+  echo "ImageMagick is not installed"
+  exit 1
+fi
+
+set -e -o pipefail
+
+OUT1="output/dashboard_states_current"
+OUT2="output/dashboard_states_other"
+DIFFOUT="output/dashboard_states_diff"
+
+mkdir -p output
+chmod o+rwx output
```

Comment:
If `output/` is created by the command running under the docker container's user (via `os.makedirs("output/dashboard_states_current")`), that directory is owned by that user and the host system user can't `mkdir -p output/dashbaord_states_diff` and vice versa. This was the only way to make that functional (short of docker quackery).